### DISCUSSION TITLE
Thank You for this!!

### DISCUSSION THREAD

**Discussion by CyberYelper** [Akascape/CTkTable#7] [General]
I just wanted to say thanks for putting the time and effort into creating this!
You have saved me from the pit of despair after many hours spent trying to find a bug that was being caused by a ttk Treeview. It was driving me insane!
Thank you! Great work - and *really* appreciated!

**Comment by Akascape** [comment]
@CyberYelper Thanks for the feedback. If you are interested in exploring other customtkinter/tkinter libraries then you can find them here:  https://github.com/Akascape/tkinter-toolkit
